### ISSUE TITLE
Fail to fetch bilibili user vedios

### ISSUE THREAD

**Issue by HughYGuo** [DIYgod/RSSHub#15592]
### Routes

```routes
/bilibili/user/video/:uid/:disableEmbed?
```

### Full routes

```fullroutes
/bilibili/user/video/39066415
```

### Related documentation

https://docs.rsshub.app/routes/social-media#bilibili

### What is expected?

The video list is fetched

### What is actually happening?

```
(node:804152) [DEP0040] DeprecationWarning: The `punycode` module is deprecated. Please use a userland alternative instead.
(Use `node --trace-deprecation ...` to show where the warning was created)
info: 🎉 RSSHub is running on port 1200! Cheers!
info: 💖 Can you help keep this open source project alive? Please sponsor 👉 https://docs.rsshub.app/sponsor
info: 🔗 Local: 👉 http://localhost:1200
info: 🔗 Network: 👉 http://10.2.48.178:1200
info: 🔗 Network: 👉 http://10.2.48.92:1200
info: <-- GET /
info: --> GET / 200 8ms
info: <-- GET /logo.png
info: --> GET /logo.png 200 2ms
info: <-- GET /bilibili/user/bangumi/39066415
info: --> GET /bilibili/user/bangumi/39066415 200 4s
info: <-- GET /bilibili/user/video/39066415
error: Error when visiting /x/space/wbi/acc/info: {"code":-352,"message":"风控校验失败","ttl":1,"data":{"v_voucher":"voucher_604a72e3-c23c-4198-aa67-df134205b505"}}
error: {"v_voucher":"voucher_3ab1a20e-b7fa-48a0-b818-53bf3195bc1d"}
error: Error in /bilibili/user/video/39066415: Error: Got error code -352 while fetching: 风控校验失败
```

### Deployment information

Self-hosted

### Deployment information (for self-hosted)

Linux debian

### Additional info

```shell
$ neofetch
       _,met$$$$$gg.          PF3L1M0A 
    ,g$$$$$$$$$$$$$$$P.       -------------- 
  ,g$$P"     """Y$$.".        OS: Debian GNU/Linux 11 (bullseye) x86_64 
 ,$$P'              `$$$.     Host: 20YRS1LB00 ThinkPad P15 Gen 2i 
',$$P       ,ggs.     `$$b:   Kernel: 6.1.0-0.deb11.7-amd64 
`d$$'     ,$P"'   .    $$$    Uptime: 22 hours, 57 mins 
 $$P      d$'     ,    $$P    Packages: 3440 (dpkg), 85 (flatpak), 7 (snap) 
 $$:      $$.   -    ,d$$'    Shell: bash 5.1.4 
 $$;      Y$b._   _,d$P'      Resolution: 2160x3840, 2160x3840 
 Y$$.    `.`"Y$$$$P"'         WM: i3 
 `$$b      "-.__              Theme: Adwaita-dark [GTK3] 
  `Y$$                        Icons: Adwaita [GTK3] 
   `Y$$.                      Terminal: x-terminal-emul 
     `$$b.                    CPU: 11th Gen Intel i7-11800H (16) @ 4.600GHz 
       `Y$$b.                 GPU: NVIDIA 01:00.0 NVIDIA Corporation Device 1fbc 
          `"Y$b._             Memory: 12323MiB / 47970MiB 
              `"""
$ node --version
v22.1.0
$ pnpm --version
9.1.1

$ git log
commit 0c36e855b50c1951b694ef9ce77c1328d9a844a1 (HEAD -> master, origin/master, origin/HEAD)
gpg: Signature made Wed 15 May 2024 12:38:49 PM JST
gpg:                using RSA key B5690EEEBB952194
gpg: Can't check signature: No public key
Author: JimenezLi <75196426+JimenezLi@users.noreply.github.com>
Date:   Wed May 15 11:38:49 2024 +0800

    fix(route): `/wechat/ershicimi` (#15570)
    
    * fix(route): /weixin/ershcimi
    
    * fix: move async function to outer scope
    
    * fix: assign return Promises to item
    
    * fix: really assign Promises to item

```

### This is not a duplicated issue

- [X] I have searched [existing issues](https://github.com/DIYgod/RSSHub/issues) to ensure this bug has not already been reported

**Comment by HughYGuo** [comment]
it looks like i forgot to pass the cookies to the url params. It works after I switched to the chromium-bundled image. Could you improve the guide in docs a little bit, on the workaround of setting cookies to url params. It looks confusing to me. Thank you in advance ❤️

**Comment by TonyRL** [comment]
Closed in 52481ce24d379a49a455bac62bc2c693dc7b7075

**Comment by Hyvi** [comment]
> Closed in [52481ce](https://github.com/DIYgod/RSSHub/commit/52481ce24d379a49a455bac62bc2c693dc7b7075)
这个没有解决，出现报错。
### 获取cookie。
![截屏2024-07-13 上午9 27 25](https://github.com/user-attachments/assets/162b1f39-dd99-45cb-a325-c5f139b9bb9f)
### 发送接口请求，失败
![截屏2024-07-13 上午9 27 46](https://github.com/user-attachments/assets/d3ddf378-0a19-4944-8dea-4ba2ebaa2f87)

### 失败的返回内容 
{"code":-412,"message":"request was banned","ttl":1}
